### PR TITLE
feat: update ftltest for injected verb clients

### PR DESCRIPTION
**Pull request by worstell**

### CODE REVIEW

File: go-runtime/ftl/ftltest/ftltest.go [hunk 1]
```diff
@@ -497,3 +498,56 @@ func WaitForSubscriptionsToComplete(ctx context.Context) {
...
+	return err
+}
+
+func call[VerbClient, Req, Resp any](ctx context.Context, req Req) (resp Resp, err error) {
+	ref := reflection.ClientRef[VerbClient]()
+	inline := server.Call[Req, Resp](ref)
+	moduleCtx := modulecontext.FromContext(ctx).CurrentContext()
+	override, err := moduleCtx.BehaviorForVerb(schema.Ref{Module: ref.Module, Name: ref.Name})
+	if err != nil {
+		return resp, fmt.Errorf("%s: %w", ref, err)
```
Comment (deniseli) [inline go-runtime/ftl/ftltest/ftltest.go:530]: nit: should we make this `"test harness failed to retrieve behavior for verb %s: %w"` and similar make L535 something like `"test harness failed to call verb %s: %w"`?

**Review by deniseli — APPROVED** [review]
This is awesome!!

**Review by alecthomas — APPROVED** [review]

**Comment by alecthomas** [comment]
Don't forget you'll need to update the customer code to use this!

**Comment by worstell** [comment]
> Don't forget you'll need to update the customer code to use this!

have been working this up today :)

File: go-runtime/ftl/ftltest/ftltest.go (hunk 1 above)
Comment (worstell) [reply go-runtime/ftl/ftltest/ftltest.go:530]: good idea!

**Comment by worstell** [comment]
think i'll wait til monday to merge this one because it will require customer changes, and i'd ideally like to couple those changes with the announcement/change for the new verb client stuff - which is much more invasive for a friday 😄

**Pull request merged**